src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java

```diff
@@ -113,20 +113,13 @@ private EmailContent constructEmailContent(EmailType emailType,
         data.put("to", email);
         data.put("company_name", companyName);
         data.put("company_number", objection.getCompanyNumber());
-        data.put("objection_id", objection.getId());
```

Comment:
Is this not needed?

Reply:
It's a dupe.

Reply:
view lines above, it was in twice, intellj highlighted it

Reply:
dupe - duplicate